spec/features/state_file/complete_intake_spec.rb

```diff
@@ -40,7 +40,7 @@
       choose "Yes"
       click_on "Continue"
 
-      expect(page).to have_text "The page with all the info from the 201"
+      expect(page).to have_text I18n.t('state_file.questions.ny_sales_use_tax.edit.title', year: "2023")
```

Comment:
Optionally, this could use `MultiTenantService.statefile.current_tax_year` (very recently added to main), but it's not clear if this project would continue to next year. I don't think there's any good reason to change this, but I just wanted to let you know in case you'd prefer it.

Reply:
Maybe it makes more sense to hardcode the year in the specs anyways

Reply:
Ooo yes good idea!